roles/validations/tasks/security/invoke_tlse_playbooks.yml

```diff
@@ -0,0 +1,11 @@
+- name: Invoke the automation playbook (dataplane certs) on controller-0
+  environment:
+    PATH: "{{ cifmw_path }}"
+  cifmw.general.ci_script:
+    executable: "/bin/bash"
+    output_dir: "{{ cifmw_basedir }}/artifacts"
+    script: |
+      set -ex -o pipefail
+      cd "{{ ansible_user_dir }}/src/gitlab.cee.redhat.com/OSP-DFG-security/automation"
```

Comment:
We should avoid leaking internal IT details like out gitlab hostname.

Reply:
This one is leaked already 

In the `openstack-k8s-operators` org:
https://github.com/search?q=org%3Aopenstack-k8s-operators%20gitlab.cee.redhat.com&type=code

In `openshift` org:
https://github.com/search?q=org%3Aopenshift%20gitlab.cee.redhat.com&type=code

Reply:
"nice" :-1: